Help Desk/helpDesk_manageDepartments.php

```diff
@@ -25,6 +27,44 @@
     $page->addError(__('You do not have access to this action.'));
 } else {
     //Proceed!
+    
+    $DepartmentGateway = $container->get(DepartmentGateway::class);
+    $SubCategoryGateway = $container->get(SubCategoryGateway::class);   
+
+    $departmentData = $DepartmentGateway->selectDepartments()->toDataSet();
+
+    $formatCategoryList = function($row) use ($SubCategoryGateway) {
+        $categories = $SubCategoryGateway->selectCategoriesByDepartment($row['departmentID'])->fetchAll();
+        if (count($categories) < 1) {
+            return __('This department does not have any subcategories.');
+        }
+        return $categories; //does this have to format as a list? if so, how do i do that
+    };
+    
+    $table = DataTable::create('departments');
+    $table->setTitle('Departments');
+
+    $table->addHeaderAction('add', __('Add'))
+            ->setURL('/modules/' . $gibbon->session->get('module') . '/helpDesk_createDepartment.php');
+
+    $table->addColumn('departmentName', __('Department Name'));
+
+    $table->addColumn('categories', __('Subcategories in department'))->format($formatCategoryList);;
+
+    $table->addActionColumn()
```

Comment:
This action column needs to be changed to work with departments, not technician groups.